cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -1159,6 +1168,16 @@ data BuildTxWith build a where
      ViewTx      ::      BuildTxWith ViewTx  a
      BuildTxWith :: a -> BuildTxWith BuildTx a
 
+instance DefaultValue (BuildTxWith ViewTx a) where
+  defaultValue = ViewTx
+
+instance DefaultValue a => DefaultValue (BuildTxWith BuildTx a) where
+  defaultValue = BuildTxWith defaultValue
```

Comment:
There is no one function that can encapsulate both `DefaultValue (BuildTxWith ViewTx a)` and `DefaultValue (BuildTxWith BuildTx a)` because we use GADTs so we use `DefaultValue` type class.

Reply:
But why would you need to? Isn't the purpose of the default value to simplify building, i.e. `BuildTx`? I personally don't see a use case for `ViewTx`, maybe you have one? If not, then a simple function `defaultTxBodyContent :: IsCardanoEra era => TxBodyContent BuildTx era` would suffice.

Reply:
Please see https://github.com/input-output-hk/cardano-node/pull/4941/files#r1126097642 for where `ViewTx` is used.